src/components/CounterBubble/CounterBubble.vue

```diff
@@ -50,9 +55,21 @@ export default {
 	name: 'CounterBubble',
 
 	props: {
-		highlighted: {
-			type: Boolean,
-			default: false,
+		type: {
+			type: String,
+			default: '',
```

Comment:
@marcoambrosini Should this maybe have a default value 'highlighted'? I think otherwise it might break if nothing is given?

Reply:
Nope, if empty the class just isn't added and the counter is in it's default state (gray background)

Reply:
But shouldn't the validator fail and produce a console warning (when using the development build)?

Reply:
you're right, I should allow the empty string in the validator, nice catch :)